guides/common/modules/proc_migrating-to-newly-deployed-satellite.adoc

```diff
@@ -0,0 +1,57 @@
+
+== Creating a backup of {ProjectServer} or {SmartProxyServer} on {RHEL} 7
+
+Before you perform a fresh installation of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 system, back up your {ProjectServer} or {SmartProxyServer} data on the {RHEL} 7 system by creating an offline backup.
+
+If you recently created an offline backup, you can perform an incremental backup to update the existing backup.
+
+*Procedure*
+
+* Perform an offline or incremental backup on the source machine:
+** To perform an offline backup, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html/administering_red_hat_satellite/backing-up-satellite-server-and-capsule-server#performing-a-full-backup-of-satellite-or-capsule_admin[Performing a Full Backup of {ProjectServer} or {SmartProxyServer}].
+** To perform an incremental backup, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html/administering_red_hat_satellite/backing-up-satellite-server-and-capsule-server#performing-an-incremental-backup_admin[Performing an Incremental Backup].
+* Transfer the backup to the target machine.
+
+== Performing a fresh install of {ProjectServer} or {SmartProxyServer} to migrate to {RHEL} 8
+
+The fresh install method has three steps:
+
+. Create a backup of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 7 system source machine.
+. Perform a fresh install of the {ProjectServer} or {SmartProxyServer} on a minimal {RHEL} 8 instance on the target machine.
+. Restore the backup on the {RHEL} 8 instance.
+
+While the backup is restoring, you can use the {RHEL} 7 system on the source machine to reduce downtime.
+However, any changes you make at this time on the {RHEL} 7 system are not in the backup.
+
+[sidebar]
+Do not perform an offline backup without Pulp content or an online backup.
+
+== Performing a fresh install of {ProjectServer} or {SmartProxyServer} on {RHEL} 8
+
+After you have created a backup of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 7 source machine, you can install {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 target machine.
+
+NOTE: A minimal {RHEL} 8 instance must be already installed on the target machine.
+
+* To install {ProjectServer}, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html/installing_satellite_server_from_a_connected_network/index[Installing {ProjectServer} from a Connected Network].
+* To install {SmartProxyServer}, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html/installing_capsule_server/index[Installing {SmartProxyServer}].
+
+
+== Restoring a backup of {ProjectServer} or {SmartProxyServer} on {RHEL} 8
+
+After you perform a fresh install of {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 instance of the target machine, you can restore the backup you previously created.
+
+*Procedure*
```

Comment:
Follow @maximiliankolb's guidance on proper modular doc layout. You should be using the template rather than making this up.